### DISCUSSION TITLE
Reset deployment exposure warning

### DISCUSSION THREAD

**Discussion by ralphotowo** [apache/airflow#26074] [General]
I have recently deployed Airflow to our Kubernetes cluster using the Helm chart. During the initial setup, Airflow was temporarily made accessible over the Internet and that resulted in the following warning:

> Recent requests have been made to /robots.txt. This indicates that this deployment may be accessible to the public internet. This warning can be disabled by setting webserver.warn_deployment_exposure=False in airflow.cfg. Read more about web deployment security [here](https://airflow.apache.org/docs/apache-airflow/2.3.0/security/webserver.html)

I have since locked down Airflow to be only accessible within our network, however I continue to see the warning above. I understand that I can set `webserver.warn_deployment_exposure=False` in Airflow's configuration file, however that will permanently disable the notice, something I'd like to avoid, just in case there's another exposure. 

<img width="1424" alt="Screenshot 2022-08-30 at 18 54 37" src="https://user-images.githubusercontent.com/1202767/187508828-d47a751e-fa47-4f61-bb81-f42d692fc32b.png">

How can I reset or clear the notice until there's another exposure?

**Comment by potiuk** [comment]
This is based on LOG table.  You can clear event with "robots" event.

**Reply by antonychumak** [reply]
can you do a little more?

**Reply by potiuk** [reply]
Ah. Mmissed that comment almost a year later, and certainly have not seen the down-thumbs. I have not realised someone might be disappointed by an answer that brings you closer to the answer.

The @Dewwi  answer is one of the ways to do it. Generally if you ever need to do something with internal DB of airflow we went to a great length to produce the ERD diagram of it, specifically for that purpose https://airflow.apache.org/docs/apache-airflow/stable/database-erd-ref.html - you can always consult it and fix it.

**Comment by Dewwi** [comment]
`DELETE FROM log where event ilike '%robot%'`

**Comment by kra5io** [comment]
DELETE FROM log WHERE event = 'robots';
